src/select/types.js

```diff
@@ -95,7 +98,7 @@ export type PropsT = {
   /** Defaults to filterOptions that excludes selected options for
    * multi select. A custom method to filter options to be displayed in the dropdown. */
   filterOptions: ?(
-    options: ValueT,
+    options: OptionsT,
```

Comment:
is this a breaking change now that flow will complain that consumers are not properly handling the case where `OptionsT` could be an object shape?

Reply:
probably not since we are making the type less strict?

Reply:
right, but if a developer does not handle the non-array case flow will complain